root.sh

```diff
@@ -107,6 +107,7 @@ else
         -Dsoversion=ON                                            \
         -Dshadowpw=OFF                                            \
         -Dvdt=ON                                                  \
+        -Dbuiltin_vdt=ON                                          \
         -DCMAKE_PREFIX_PATH="$FREETYPE_ROOT;$SYS_OPENSSL_ROOT;$GSL_ROOT;$ALIEN_RUNTIME_ROOT;$PYTHON_ROOT;$PYTHON_MODULES_ROOT"
   FEATURES="builtin_pcre mathmore xml ssl opengl minuit2 http
             pythia6 roofit soversion vdt ${CXX11:+cxx11} ${CXX14:+cxx14} ${XROOTD_ROOT:+xrootd}
```

Comment:
Hi @dberzano ,

Thanks for reviewing this PR. I assume you're referring to the line
`FEATURES="builtin_pcre ...`.

Building root5 with `-Dvdt=ON` and `-Dbuiltin_vdt=ON`, the `-Dbuiltin_vdt=ON` is simply ignored. Consequently, with root5, `root-config --features` gives `vdt` but not `builtin_vdt`. 
With root6, `root-config --features` gives both `vdt` and `builtin_vdt`.

Checking for `builtin_vdt` in `FEATURES="...` will fail with root5.

Cheers,
Hans

root 5.34/30
`root-config --features`
alien asimage astiff builtin_afterimage builtin_ftgl builtin_glew builtin_pcre builtin_lzma cxx11 cintex exceptions explicitlink fftw3 fortran genvector http mathmore memstat minuit2 monalisa opengl pythia6 pythia6_nolink python reflex roofit shared soversion ssl thread tmva **vdt** xft xml x11 xrootd

root 6.10/00
`root-config --features`
alien asimage astiff builtin_afterimage builtin_ftgl builtin_gl2ps builtin_glew builtin_llvm builtin_lzma builtin_pcre builtin_tbb builtin_unuran **builtin_vdt** cling cxx11 exceptions explicitlink fftw3 fortran genvector http imt mathmore memstat minuit2 monalisa opengl pch pythia6 pythia6_nolink python roofit shared soversion ssl thread tmva **vdt** x11 xft xml xrootd